rust_snuba/src/processors/metrics_summaries.rs

```diff
@@ -158,4 +169,48 @@ mod tests {
         process_message(payload, meta, &ProcessorConfig::default())
             .expect("The message should be processed");
     }
+
+    #[test]
+    fn test_missing_values() {
+        let span = br#"{
```

Comment:
i think if you have a testcase like this it would be best in sentry-kafka-schemas, then it will be implicitly tested